ui/pages/confirmations/hooks/useTypedSignSignatureInfo.test.js

```diff
@@ -0,0 +1,20 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import { TokenStandard } from '../../../../shared/constants/transaction';
+import { permitNFTSignatureMsg } from '../../../../test/data/confirmations/typed_sign';
+import { TypedSignSignaturePrimaryTypes } from '../constants';
+import { useTypedSignSignatureInfo } from './useTypedSignSignatureInfo';
+
+describe('useTypedSignSignatureInfo', () => {
+  describe('isNftTransfer', () => {
+    it('should return false if transaction is not NFT transfer', () => {
```

Comment:
Minor, is it worth unit tests for the early return conditions?

Reply:
yep, I added one test for that